tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -226,7 +225,7 @@ def check_roslaunch(f):
     # check for missing nodes
     for pkg, node_type in nodes:
         try:
-            if not roslaunch.resource_cache.find_node(pkg, node_type):
+            if not roslib.packages.find_node(pkg, node_type):
```

Comment:
This is easy since the calling scope already contains a rospack instance: 47f6c58931acda0f44cd5cb29395596500cc58c6